armory/scenarios/base.py

```diff
@@ -208,7 +208,7 @@ def run_config(config_json, from_file=False, check=False, mongo_host=None):
         "--mongo",
         dest="mongo_host",
         default=None,
-        help="Send scenario results to a MongoDB instance at the given host (eg 'localhost', '1.2.3.4', 'mongodb://USER:PASS@5.6.7.8')",
+        help="Send scenario results to a MongoDB instance at the given host (eg mongodb://USER:PASS@5.6.7.8')",
```

Comment:
Why are we removing the other types of instances?

Also, need to add a "'":
`        help="Send scenario results to a MongoDB instance at the given host (eg 'mongodb://USER:PASS@5.6.7.8')",`

Reply:
Those were removed when we added authentication to the Mongo server, as the name must be passed with the schema in order for the username and password to be supplied